pdns/dnsdist.cc

```diff
@@ -1009,6 +1009,9 @@ bool processQuery(LocalHolders& holders, DNSQuestion& dq, string& poolname, int*
       case DNSAction::Action::Delay:
         *delayMsec = static_cast<int>(pdns_stou(ruleresult)); // sorry
         break;
+      case DNSAction::Action::SetTempFailureTTL:
```

Comment:
If we make `tempFailureTTL` a member of `DNSQuestion` we don't need to add a new `DNSAction::Action` value, `TempFailureTTLAction` can just set the value the way `SkipCacheAction` does.